lib/compat/wordpress-6.2/class-wp-theme-json-6-2.php

```diff
@@ -0,0 +1,219 @@
+<?php
+/**
+ * WP_Theme_JSON_6_2 class
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Class that encapsulates the processing of structures that adhere to the theme.json spec.
+ *
+ * This class is for internal core usage and is not supposed to be used by extenders (plugins and/or themes).
+ * This is a low-level API that may need to do breaking changes. Please,
+ * use get_global_settings, get_global_styles, and get_global_stylesheet instead.
+ *
+ * @access private
+ */
+class WP_Theme_JSON_6_2 extends WP_Theme_JSON_6_1 {
+	const APPEARANCE_TOOLS_OPT_INS = array(
+		array( 'border', 'color' ),
+		array( 'border', 'radius' ),
+		array( 'border', 'style' ),
+		array( 'border', 'width' ),
+		array( 'dimensions', 'minHeight' ),
+		array( 'color', 'link' ),
```

Comment:
Extreme OCD nit: Should we maintain the alphabetical ordering for the rest of this array? Keep the sorting of the other config arrays going as well?

Reply:
I love a good nit like this, cheers, I've updated the order 👍